workflow-controller/controller-service/src/main/java/io/hyscale/controller/validator/ClusterValidator.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.controller.validator;
+
+import io.hyscale.commons.validator.Validator;
+import io.hyscale.controller.model.WorkflowContext;
+
+public interface ClusterValidator extends Validator<WorkflowContext>{
+
+}
```

Comment:
Why interface here ? 
One validator across all the commands and the same input.